### PR TITLE
[pkgconfig] Add pkgconfig support, and fix devel package. MER#1837

### PR DESCRIPTION
**Pull request by abranson**
Devel packages was a bit broken before, and gst-droid could not be built against by other packages. This fixes it and adds a pkgconfig file to make it easier. This is in preparation for gst-omx, but doesn't actually change the functionality of gst-droid.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -167,15 +167,16 @@ AC_DEFINE_UNQUOTED(GST_API_VERSION, "$GST_API_VERSION",
...
+        common/Makefile
+        common/m4/Makefile
+        gst-libs/Makefile
+        gst-libs/gst/Makefile
+        gst-libs/gst/droid/Makefile
+        gst/Makefile
+        gst/droideglsink/Makefile
+        gst/droidcodec/Makefile
+        gst/droidcamsrc/Makefile
+        tools/Makefile])
```
Comment (sledges) [inline configure.ac]: unnecessary diff-polluting hunk (see supporting comment beloow)

File: gst/Makefile.am [hunk 2]
```diff
@@ -6,8 +6,8 @@ plugin_LTLIBRARIES = libgstdroid.la
 
 libgstdroid_la_CFLAGS = $(GST_CFLAGS) $(NGI_CFLAGS) $(EGL_CFLAGS) \
                         $(NGM_CFLAGS)\
-	                -I$(top_builddir)/gst-libs/ \
-			-I/usr/include/droidmedia/ \
+                        -I$(top_builddir)/gst-libs/ \
+                        -I/usr/include/droidmedia/ \
                         -Idroidcamsrc/ \
```
Comment (sledges) [inline gst/Makefile.am]: if you would like to fix whitespace, I'd suggest to create a separate "cleanup" commit (without tag and bugref)

**Review by sledges — CHANGES_REQUESTED** [review]
whitespace nits :)

File: gst/Makefile.am (hunk 2 above)
Comment (abranson) [reply gst/Makefile.am]: I'll just leave it for now.

Comment (sledges) [reply gst/Makefile.am]: It's just tab-space conversion, in vim shows up fine with ts=8, and github borks it due to diff :)

**Review by sledges — APPROVED** [review]
LGTM from code perspective, unsure about autotools semantics. If it builds it's good enough, if you can't get anyone else to review this

**Comment by abranson** [comment]
Thanks, I'll get it in now I think so there's only omx left.
